tests/integration/test_fim/test_registry/test_registry_basic_usage/test_basic_usage_registry_duplicated_entries.py

```diff
@@ -0,0 +1,112 @@
+import os
+
+import pytest
+from wazuh_testing import global_parameters
+import wazuh_testing.fim as fim
```

Comment:
```suggestion
import wazuh_testing.fim 
```

If we are importing `fim` with the same name, this is not necessary